core/services/aws/bloodDonation/donorSearchEvaluator.ts

```diff
@@ -0,0 +1,28 @@
+interface DonorSearchInput {
```

Comment:
any reason why this is interface

Reply:
vai, in all lambda we are using an interface to take input